backend/lucene/src/main/java/org/hibernate/search/backend/lucene/types/sort/impl/AbstractLuceneStandardFieldSortBuilder.java

```diff
@@ -49,11 +50,11 @@
 	protected Object missingValue;
 
 	protected AbstractLuceneStandardFieldSortBuilder(
-			LuceneSearchContext searchContext,
-			String absoluteFieldPath, String nestedDocumentPath,
-			DslConverter<?, ? extends F> converter, DslConverter<F, ? extends F> rawConverter,
-			LuceneCompatibilityChecker converterChecker, C codec,
-			Object sortMissingValueFirstPlaceholder, Object sortMissingValueLastPlaceholder) {
+		LuceneSearchContext searchContext,
+		String absoluteFieldPath, String nestedDocumentPath,
+		DslConverter<?, ? extends F> converter, DslConverter<F, ? extends F> rawConverter,
+		LuceneCompatibilityChecker converterChecker, C codec,
+		Object sortMissingValueFirstPlaceholder, Object sortMissingValueLastPlaceholder) {
```

Comment:
While I can imagine why you prefer to indent code this way, this does not fit the style guide of this project. Please don't reindent code like this.

You can find information on how to set up your IDE on our website: [here for Eclipse](https://hibernate.org/community/contribute/eclipse-ide/) and [here for IDEA](https://hibernate.org/community/contribute/intellij-idea/). I do not know about netbeans, though (that's what you are using, right?)

Reply:
Little Netbeans is in some cases automated. It helps a lot while programming, but unfortunately it doesn't give 100% compatibility with Eclipse. It's hard to see what a formatter will do, I rely on "maven-checkstyle-plugin". The differences are not intentional. Result from auto text formatting.

Reply:
I found a parameter. Corrected.

Reply:
Corrected